src/software/geom/rectangle.cpp

```diff
@@ -11,8 +11,8 @@ bool Rectangle::expand(double amount)
     {
         return false;
     }
-    Point add(amount, amount);
-    min_corner -= add;
+    Vector add(amount, amount);
+    min_corner += -add;
```

Comment:
why not `-=` here?

Reply:
Added a -= operator, fixed in latest update